assume/markets/clearing_algorithms/complex_clearing.py

```diff
@@ -217,11 +217,16 @@ def primal_eql_dual(m):
     eps = 1e-4
     solvers = check_available_solvers(*SOLVERS)
 
-    solver = SolverFactory(solvers[0])
+    solver = (
+        SolverFactory("gurobi") if "gurobi" in solvers else SolverFactory(solvers[0])
```

Comment:
If you want to prefer gurobi, you can reorder the list of Solvers at the top of the file ;)
I think it makes sense to prioritize commercial solvers if you have them available anyway.